src/Defra.PTS.Web.UI/Views/Shared/_FooterPartial.cshtml

```diff
@@ -30,21 +30,20 @@
                     @if(Thread.CurrentThread.CurrentCulture.EnglishName != "Welsh")
                     {
                         <li class="govuk-footer__inline-list-item">
-                            <a class="govuk-footer__link" href="" id="localeCy">
+                            <a class="govuk-footer__link" href="#" id="localeCy" onclick="changeLanguage('cy')">
```

Comment:
this is done in jQuery, it does not need to be done here.

Reply:
When clicked, it triggers the changeLanguage function with the argument 'cy', which stands for Welsh language.